src/logger.js

```diff
@@ -0,0 +1,13 @@
+import Config from './config'
+
+export default class Logger {
+  debug(message) {
+    if (Config.debug) {
+      console.log(message)
```

Comment:
what if we used `console.info` and had a `Logger.info` method that called that?  and then we could save `console.log` for regular logging usage?

Reply:
@jmartin-sul as a friendly amendment, how about using `console.debug()` here? (I didn't know `console.info/debug/etc.` existed!)

Reply:
ah, yeah, so i went looking to see if AWS supported those other flavors (i hadn't realized console logging automatically got shunted to cloud watch till you mentioned it a couple weeks ago), and what i found seemed to indicate that lambadas don't support `debug`, though i didn't find any info specific to fargate.  but yeah, my first inclination was toward `debug`, and the uncertainty around support was the only reason i didn't suggest it.  happy to give it a shot and see what happens and adjust if need be.  not a huge codebase, so wouldn't be too onerous a find/replace.

(and now of course i'm having trouble finding the documentation i was looking at last night)

Reply:
since it's there now, and preferable if it works, let's give `debug` a shot?  merging...

Reply:
@mjgiarlo this is what i was looking at:  https://docs.aws.amazon.com/lambda/latest/dg/nodejs-prog-model-logging.html
>  Your Lambda function can contain logging statements. AWS Lambda writes these logs to CloudWatch. If you use the Lambda console to invoke your Lambda function, the console displays the same logs.
> 
> The following Node.js statements generate log entries:
>  * console.log()
>  * console.error()
>  * console.warn()
>  * console.info()